### PR TITLE
Create an example for random

### PR DESCRIPTION
**Pull request by ranya-cfa**

### CODE REVIEW

File: examples/random/main.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+static POPULATION: u64 = 10;
+
+define_rng!(MyRng);
+
+fn main() {
+    let mut context = Context::new();
+    context.init_random(SEED);
+
+    let random_person = context.sample_range(MyRng, 0..POPULATION);
+    let person_id = random_person;
```
Comment (ChiragKumar9) [inline examples/random/main.rs:16]: I believe this line is residual from when this was being "stringed"

File: examples/random/main.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        move |context| {
+            println!(
+                "Person {} was infected at time {}",
+                person_id,
+                context.get_current_time()
+            );
+        }
+    });
+
+    let recovery_time: f64 = context.sample_distr(MyRng, Uniform::new(2.0, 10.0));
```
Comment (ChiragKumar9) [inline examples/random/main.rs:28]: I wonder if it wouldn't hurt to put another example here of using the `sample_bool` method -- something like if you draw some bool with probability p that is true, the recovery time is drawn from Uniform(2, 10), but if it is false, the recovery time is drawn from some custom PDF using the sample function or just Exp(1) if we want it to be more straightforward. I just think it's nice to give people examples, even if the method is straightforward to use in this early stage.

File: examples/random/main.rs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    context.add_plan(1.0, {
```
Comment (k88hudson-cfa) [inline examples/random/main.rs:29]: How about make the plan execute at `recovery_time` and then log `context.get_current_time()`?

File: examples/random/main.rs (hunk 2 above)
Comment (k88hudson-cfa) [reply examples/random/main.rs:28]: I agree, let's work on integrating some more example usages

**Review by k88hudson-cfa — CHANGES_REQUESTED** [review]

**Review by k88hudson-cfa — APPROVED** [review]

**Pull request merged**